### PR TITLE
APPS-544 - don't create default value for file-type outputs

### PR DESCRIPTION
**Pull request by jdidion**
If an output parameter has expression that is a File, Directory or collection thereof, the paths will be local to the worker so we cannot use them as the default value (since file inputs need to be given as links to dx files).

This PR also activates some additional integration tests and removes unused tests.

### CODE REVIEW

**Review by commandlinegirl — DISMISSED** [review]
LGTM

File: scripts/build_all_releases.sh [hunk 1]
```diff
@@ -101,6 +101,7 @@ function usage_die
     echo "  --production-token <string>: an auth token for the production environment"
     echo "  --docker-user <string>: docker user name"
     echo "  --docker-password <string>: docker password"
+    echo "  --branch <string>: branch to build from (default=main)"
```
Comment (commandlinegirl) [inline scripts/build_all_releases.sh:104]: Thanks for this!

**Review by commandlinegirl — CHANGES_REQUESTED** [review]
Minor

File: build.sbt [hunk 2]
```diff
@@ -117,10 +117,13 @@ val executorCwl = project
 
 // DEPENDENCIES
 
+val githubResolver = Resolver.githubPackages("dnanexus", "dxScala")
+resolvers += githubResolver
```
[the remark this answers is not in the document]
Comment (jdidion) [reply build.sbt:121]: it is used to resolve snapshot packages stored in GitHub packages

**Review by commandlinegirl — APPROVED** [review]

**Pull request merged**